components/softdevice/s115/s115_API/include/ble_ranges.h

```diff
@@ -67,13 +67,13 @@ extern "C" {
 #define BLE_SVC_LAST           0x6B       /**< Common BLE SVC last. */
 
 #define BLE_GAP_SVC_BASE       0x6C       /**< GAP BLE SVC base. */
-#define BLE_GAP_SVC_LAST       0x9B       /**< GAP BLE SVC last. */
+#define BLE_GAP_SVC_LAST       0xA0       /**< GAP BLE SVC last. */
 
-#define BLE_GATTC_SVC_BASE     0x9C       /**< GATTC BLE SVC base. */
-#define BLE_GATTC_SVC_LAST     0xA7       /**< GATTC BLE SVC last. */
+#define BLE_GATTC_SVC_BASE     0xA1       /**< GATTC BLE SVC base. */
+#define BLE_GATTC_SVC_LAST     0xAC       /**< GATTC BLE SVC last. */
 
-#define BLE_GATTS_SVC_BASE     0xA8       /**< GATTS BLE SVC base. */
-#define BLE_GATTS_SVC_LAST     0xB7       /**< GATTS BLE SVC last. */
+#define BLE_GATTS_SVC_BASE     0xAD       /**< GATTS BLE SVC base. */
+#define BLE_GATTS_SVC_LAST     0xBC       /**< GATTS BLE SVC last. */
```

Comment:
These changes makes it not compatible with previous SoftDevice hex. There should be a changelog entry for this, or am I missing something?

Reply:
Alright